server/sonar-server/src/main/java/org/sonar/ce/settings/ProjectConfigurationFactory.java

```diff
@@ -20,25 +20,27 @@
 package org.sonar.ce.settings;
 
 import org.sonar.api.ce.ComputeEngineSide;
+import org.sonar.api.config.Configuration;
 import org.sonar.api.config.Settings;
+import org.sonar.api.config.internal.ConfigurationBridge;
 import org.sonar.db.DbClient;
 
 @ComputeEngineSide
-public class ProjectSettingsFactory {
+public class ProjectConfigurationFactory {
 
   private final Settings globalSettings;
   private final DbClient dbClient;
 
-  public ProjectSettingsFactory(Settings globalSettings, DbClient dbClient) {
+  public ProjectConfigurationFactory(Settings globalSettings, DbClient dbClient) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)